### PR TITLE
Create Unit test project for thermometer

### PR DESCRIPTION
**Pull request by chenghuang-mdsol**
Create Unit test project for thermometer project. Not sure about the naming conventions on the project name, though

### CODE REVIEW

**Comment by art2003** [comment]
Should we have App.config.template under source control?

File: Medidata.Cloud.Thermometer.UnitTests/LiteRestAppTests.cs [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+                        var readTask = response.Content.ReadAsStringAsync();
+                        readTask.Wait();
+                        var content = readTask.Result;
+                        Assert.IsTrue(content.Contains("abc result"));
+                    }
+                    else
+                    {
+                        Assert.Fail("404 Not Found");
+                    }
+                }
```
Comment (art2003) [inline Medidata.Cloud.Thermometer.UnitTests/LiteRestAppTests.cs]: Can we break it in two tests

File: Medidata.Cloud.Thermometer.UnitTests/LiteRestAppTests.cs [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+                    {
+                        var readTask = response.Content.ReadAsStringAsync();
+                        readTask.Wait();
+                        var content = readTask.Result;
+                        Assert.IsTrue(content.Contains("abc result"));
+                    }
+                    else
+                    {
+                        Assert.Fail("404 Not Found");
+                    }
```
Comment (art2003) [inline Medidata.Cloud.Thermometer.UnitTests/LiteRestAppTests.cs]: same as above

File: Medidata.Cloud.Thermometer.UnitTests/LiteRestAppTests.cs [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+                client.Stop();
+
+            };
+            client.Start();
+            client.WaitForConnection();
+            client.PushMessage(new ThermometerResult(){Message = "Status"});
+            //client.Stop();
+            Thread.Sleep(1000);
+            // Validate the server's signature string 
+
```
Comment (art2003) [inline Medidata.Cloud.Thermometer.UnitTests/LiteRestAppTests.cs]: Where is Assertion part

**Comment by junshao** [comment]
I think we should keep these changes in a separate PR and not merge into the "master" branch. Because they are pre-investigation and tech-comparison before we decide what the Thermometer should be like.

**Comment by junshao** [comment]
@chenghuang-mdsol Can you please close this PR against the master, and push directly to a new branch like "junshao/experimental/framework_comparison"? Thanks.

**Pull request closed without merging**